README.md

```diff
@@ -1,29 +1,96 @@
-## Develop
-```bash
-yarn dev
-# or `npm run dev`
-```
-
----
-## Build
-```bash
-yarn build
-# or `npm run build`
-```
-
----
-<p align="center">
-<img src="./powered-by-vitawind-bright.png">
-</p>
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Analitycs Reporting ](#-analitycs-reporting-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👤 Author ](#-author-)
+  - [✅ Current Features ](#-current-features-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Analitycs Reporting <a name="about-project"></a>
+
+**Analitycs Reporting**, is a web application built using React that allows me to view the performance of different website I have. The app fetches data from Airtable which in turn takes data from various visitor data sources allowing me to see a summary of views, active users and bounce rates of my sites.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | JS | GIT
+
+Vite | Tailwind | React | Redux | GH pages | Jest | fontAwesome
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+On Render
+- [Live demo](https://wickathou.github.io/analytics-reporting/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+GIT | A code editor | A web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  `git clone git@github.com:wickathou/analytics-reporting.git`
+
+### Usage
+
+To run the project, use the command
+`npm run dev`
+
+To run the tests, use the command
+`npm run test`
+
+## 👤 Author <a name="author"></a>
+
+**Javier Hernandez**
+
+- GitHub: [@wickathou](https://github.com/wickathou)
+- LinkedIn: [@javierjhm](https://linkedin.com/in/javierjhm)
+- Portfolio: [End Design Co.](https://works.enddesign.co/)
+
+## ✅ Current Features <a name="current-features"></a>
+
+- [Done] **[Visual interface for App]**
+- [Done] **[Category filtering system]**
+- [Done] **[Separate webapp views per site managed]**
+- [Done] **[Unit test with Jest and Testing Library]**
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve visual design]**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check the [issues page](https://github.com/wickathou/analytics-reporting/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to fork it or star it
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks Microverse, learn how to code > [Join Microverse](https://www.microverse.org/?grsf=9m3hq6)
```

Comment:
Kindly ensure that your readme file contains and "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).